BrainPortal/app/views/groups/_groups_table.html.erb

```diff
@@ -124,7 +124,7 @@
           <h4><%= h crop_text_to(25, g.name) %></h4>
           <hr>
           <span class="project_user_count">
-            <% if user_count > 1 %>
+            <% if g.users.include?(current_user) && user_count > 1 %>
```

Comment:
Explanation?

Reply:
I only want a User to have access to the count of `Users` if this user are not a member of this groups. For example I do not want a user to have acces to this information if he see this project because it is a public one.